cpp/bridges_module/bridges_test.cpp

```diff
@@ -0,0 +1,272 @@
+#include <chrono>
+#include <queue>
+#include <random>
+
+#include <gtest/gtest.h>
+#include <mg_graph.hpp>
+
+#include "algorithm/bridges.hpp"
+
+/// This class is threadsafe
+class Timer {
+ public:
+  Timer() : start_time_(std::chrono::steady_clock::now()) {}
+
+  template <typename TDuration = std::chrono::duration<double>>
+  TDuration Elapsed() const {
+    return std::chrono::duration_cast<TDuration>(std::chrono::steady_clock::now() - start_time_);
+  }
+
+ private:
+  std::chrono::steady_clock::time_point start_time_;
+};
```

Comment:
This can be put into utils or maybe even create test_utils. :thinking:

Reply:
This is a good idea, might be useful to have test utils for creating test graphs and having a timer.